### ISSUE TITLE
Re-add Blindsight Luminous range + Movement change

### ISSUE THREAD

**Issue by neoNoteCoder** [nekomaster1000/Infernal-Expansion#193]
**Describe the addition:**

- Previously, the Blindsights had an increased detection range whenever you got luminous near them. This made the luminous effect more dangerous for the player to have. Since this has been removed, Blindsights are barely a threat anymore, since they don't actively chase the player. I believe the old Blindsight luminous range being re-added would help the Blindsights become a more versatile enemy. (Only if you have luminous, of course)

- Furthermore, Blindsights used to jump around at a far range, which would make them a very threatening enemy to come across. They are now stationary mobs, making them less of a threat. Instead of making far, long jumps, they could make shorter and slower jumps to move around the biome. A 4-second interval between jumps would also help to balance out their movement patterns.

**Comment by Dariensg** [comment]
I have a couple questions for clarification, and some explanation of why changes were made.

- The Blindsight never had a further detection range when the player has Luminous. The Blindsights are meant to have terrible vision, and even Luminous wouldn't help with that, especially with the bright biome they live in, light would not help them to distinguish prey in my opinion. We did reduce the general range of the Blindsight, mostly due to complaints that they would lock onto you from fairly far away and attack. I could see increasing that range a bit, but I'm not sure that a "Luminous Detection Range" is the way to do that. I think it should be a general increase if at all.
- At this point, Blindsights are only stationary until you make the mistake of walking into their range. Once a Blindsight has identified a target, they still will hop at you fairly quickly. These jumps really make the Blindsight terrifying since pillaring up won't do much to deter them. And, if they hit you, the Luminous effect they inflict will cause any nearby Glowsquitos to swarm you as well. I feel that the Blindsights, in this way, become more of an obstacle that you should try to avoid and will punish you if you don't than a moving enemy that could chase after you at any point in time. I feel like we've tried to make the GSC a place where most of the dangers are avoidable if you're careful, but, if not, they can be very deadly.

As for why we made these changes, we actually got several complaints that the Blindsights were too aggressive and dangerous. They'd hop around and find you and chase after you. Their movement allowed them to hop into range of you at any point and begin chasing you down. Our solution was to reduce that range and also make them sit still unless chasing something. Additionally, there were issues with them hopping around and wiping out the entire Glowsquito population. I think a good balance is to have them more like a Flounder, sitting in wait on the ground for prey to make the mistake of coming near them before they leap out in attack.

Let me know how you feel about all of this!
